src/Bounds.cpp

```diff
@@ -2845,7 +2967,7 @@ map<string, Box> boxes_touched(const Expr &e, Stmt s, bool consider_calls, bool
             debug(0) << "  " << j << ": " << it.second[j].min
                      << " .. "
                      << it.second[j].max
-                     << "\n";
+                     << '\n';
```

Comment:
I dug into benchmarking `<< '\n'` vs `<< "\n"` at one time, and weirdly enough, `<< "\n"` is faster. Sending a single character to a stream goes through a weird slow path.

Reply:
(in libstdc++)

Reply:
Done